src/modules/fetchAPI.js

```diff
@@ -0,0 +1,40 @@
+/* eslint-disable no-unused-vars */
+const requestURL = 'https://us-central1-js-capstone-backend.cloudfunctions.net/api/games/viIYUtZdNuIa0PPWUYwJ/scores/';
+
+const player = document.getElementById('name');
+const playerScore = document.getElementById('score');
+
+export const post = async () => {
+  const res = await fetch(requestURL, {
+    method: 'POST',
```

Comment:
- [OPTIONAL] In my opinion, `res` is an unused variable, so you can remove it 👌💯
- [OPTIONAL] I think now you don't have to disable the linter `/* eslint-disable no-unused-vars */`

Reply:
ok, will do so!